test/unit/uk/gov/hmrc/exports/services/mapping/declaration/BorderTransportMeansBuilderSpec.scala

```diff
@@ -77,12 +78,21 @@ class BorderTransportMeansBuilderSpec extends WordSpec with Matchers with Mockit
       }
 
       "border transport only" in {
-        val model = aDeclaration(withoutBorderTransport(), withDepartureTransport(borderModeOfTransportCode = "code"))
+        val model = aDeclaration(withoutBorderTransport(), withDepartureTransport(ModeOfTransportCode.Maritime))
+        val declaration = new Declaration()
+
+        builder.buildThenAdd(model, declaration)
+
+        declaration.getBorderTransportMeans.getModeCode.getValue should be("1")
+      }
+
+      "border transport is Empty" in {
```

Comment:
I can see you have a test for Empty here.  I need to go back and work out how it passes!  :-)